custom_components/resmed_myair/redaction.py

```
@@ -0,0 +1,34 @@
+"""Shared redaction helpers for ResMed myAir logs."""
+
+from collections.abc import Mapping
+from typing import Any
+
+from .const import KEYS_TO_REDACT
+
+REDACTED = "**REDACTED**"
+
+
+def redact_dict(data: Any | None) -> Any | None:
```

Ruff (0.15.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `data`

(ANN401)